components/tikv_util/src/time.rs

```diff
@@ -321,12 +363,28 @@ impl Instant {
         }
     }
 
+    pub(crate) fn saturating_elapsed_duration(later: Timespec, earlier: Timespec) -> Duration {
+        if later >= earlier {
+            (later - earlier).to_std().unwrap()
+        } else {
+            debug!(
```

Comment:
Since this case should happen rarely, the log here should be an error or warn?

Reply:
Good idea.